include/ceed-impl.h

```diff
@@ -349,6 +349,7 @@ struct CeedOperator_private {
   CeedInt                   num_context_labels;
   CeedInt                   max_context_labels;
   CeedContextFieldLabel    *context_labels;
+  CeedBasis                *active_basis_in, *active_basis_out;
```

Comment:
I'm belatedly realizing that we probably want this as part of 

```
struct CeedOperatorAssemblyData_private {
  Ceed          ceed;
  CeedInt       num_active_in, num_active_out;
  CeedEvalMode *eval_modes_in, *eval_modes_out;
  CeedScalar   *B_in, *B_out;
  CeedBasis     *bases_in, bases_out;
  CeedElemRestriction *elem_restrs_in, *elem_restrs_out;
};
```

Reply:
Yeah, so you mean make `basis_in/out` to array and use this function `CeedOperatorAssemblyDataCreate`?
It seems we get the active basis there.